services/gnome-extensions/gnome-extensions-downloads.service.js

```diff
@@ -0,0 +1,37 @@
+import { renderDownloadsBadge } from '../downloads.js'
+import { pathParams } from '../index.js'
+import GnomeExtensionsBase from './gnome-extensions-base.js'
+
+export default class GnomeExtensionsDownloads extends GnomeExtensionsBase {
+  static category = 'downloads'
+
+  static route = {
+    base: 'gnome-extensions/downloads',
```

Comment:
We have older badges that follow a different format, but `dt` would align better with our [guidelines](https://github.com/badges/shields/blob/master/doc/badge-urls.md).